modules/java/pages/gs-prereqs.adoc

```diff
@@ -37,60 +39,63 @@ So, If you have not already done so, see {sgw-prepare--xref}
 * Within {svr}, create a bucket named {nmSampleAppDb}
 * Create a RBAC user for {sg} with username = `sync-gateway` and password = `password`
 
-== Downloaded Binaries
 
-===  Package Contents
+== MacOS and Windows
 
-The download package contains a license file, jar libraries for the appropriate edition of the {cbl} and a set of Linux _shared libraries_.
+You may now proceed directly to xref:gs-install.adoc[].
 
-Get the download package from here -- {downloads-mobile--xref}.
+[#additional-steps-for-linux]
+== Additional Steps For Linux
 
-* Community Edition -- {packageNmCE}
-* Enterprise Edition -- {packageNmEE}
-
-When unpacked the package contains the following:
-
-* A `lib` folder containing all necessary JAR files:
-** {packageNmCE} or for EE {packageNmEE}
-** okhttp-3.14.7.jar
-** okio-1.17.2.jar
-
-* A `support` folder containing the Linux native libraries:
-+
-NOTE: This means you do not need to download and-or build compatible versions of system libraries for the Linux platform of choice.
-
-** `libz` (requires zlib v.1.2.11+)
-** `lib{cpp}` requires libc++ v.3.6.0+)
-** `libicu` (requires ICU4C v.5.4.1+)
-
-// Go back to <<GET STARTED>>
+Before proceeding to xref:gs-install.adoc[], you will need to setup your build environment, and make the supplied support libraries available.
 
 === Steps
 
-. Download the _zip_ file from here -- {downloads-mobile--xref}.
-. Unpack the downloaded file to a location accessible to -- and usable by -- your chosen *build* environment.
-+
-We'll refer to that location -- `<your dir>/{packageNmCE}` -- as the `<pathToCbl>`.
+. Download the _zip_ file from here -- https://packages.couchbase.com/releases/couchbase-lite-java/{version-maintenance-java}/couchbase-lite-java-linux-supportlibs-{version-maintenance-java}.zip
+.
+. Unpack the downloaded file to a location accessible to -- and usable by -- your chosen *build* environment, for example `your_dir/couchbase-lite-java-{version-maintenance-java}`.
 
-. Include the following dependency in your `build.gradle` file, you can remove any {cbl} Maven references:
+. Include the following dependency in your `build.gradle` file:
 +
+[source,gradle,subs="attributes"]
 ----
 Dependencies {
-  implementation fileTree(include: ['*.jar'], dir: <pathToCbl>/lib>
+  implementation fileTree(include: ['*.jar'], dir: your_dir/couchbase-lite-java-{version-maintenance-java})
```

Comment:
I think I see what is going on, here.  This used to be a reference to an exploded zip.  It should now be a reference to a maven object.  That requires two things:

in setting.gradle (formerly in build.gradle: maybe some people will still have it there) you need to identify our maven server:
`
repositories {
    ...
    maven { url "http://proget.build.couchbase.com/maven2/internalmaven" }
   ...
}
`

Next, replacing the line above, in the dependencies (note lower case!) section in build.gradle you need something like this: 
`
dependencies {
    ...
    implementation 'com.couchbase.lite:couchbase-lite-java-ee:3.1.3'
    ...
}
`
The "ee" is the enterprise edition which people should not be using unless they have a license.  The "3.1.3" is, I think, your "version-maintenance-java}

Reply:
FWIW, to use the CE (community edition) the repositories section just needs `mavenCentral()` (which is probably already there) instead of that `url` line.